addons/website_forum/populate/mail_message.py

```diff
@@ -0,0 +1,44 @@
+from odoo import models
+from odoo.tools import populate
+
+CP_WEIGHTS = { '1': 35, '2': 30, '3': 25, '4': 10 }
+MAX_COMMENT = 4
+COMMENTS_RANGE = list(range(1,(MAX_COMMENT + 1)))
+WEIGHTS = list(CP_WEIGHTS.values())
+
+class Message(models.Model):
+    _inherit = "mail.message"
+    _populate_dependencies = ["forum.post"]
+    _populate_sizes = {'small': 100, 'medium': 1000, 'large': 90000 }
+
+    def _populate_factories(self):
+        post_ids = self.env.registry.populated_models['forum.post']
+        question_ids = self.env['forum.post'].search(
+            [('id', 'in', post_ids), ('parent_id', '=', None)]).mapped('id')
+        answer_ids = self.env['forum.post'].search(
+            [('id', 'in', post_ids), ('parent_id', '!=', None)]).mapped('id')
+        params = {
+            "comments_per_post": 0,
+            "post_type": 'q',
+            "post_id": None,
+        }
```

Comment:
I'm not sure this is clearer than 
```suggestion
        nb_comments = 0
        is_question = True
        post_to_comment_id = None
```